### PR TITLE
reworked controlled vocab language keys

### LINKED ISSUE #10826: Metadata Blocks Properties check failing after merging #10762, Missing key 'controlledvocabulary.language.abkhaz
Ever since we merged #10762 we've seen failures of the "Metadata Blocks Properties" check:

Screenshot 2024-09-06 at 9 56 06 AM

It shows, for example, `Error: Missing key 'controlledvocabulary.language.abkhaz=...' in /home/runner/work/dataverse/dataverse/src/main/java/propertyFiles/citation.properties` at https://github.com/IQSS/dataverse/actions/runs/10703509353/job/29674245185

Screenshot 2024-09-06 at 9 58 35 AM

The check was added in this PR:

- #9176

### PR DESCRIPTION
**Pull request by stevenwinship**
**What this PR does / why we need it**: Ever since we merged https://github.com/IQSS/dataverse/pull/10762 we've seen failures of the "Metadata Blocks Properties" check

**Which issue(s) this PR closes**:https://github.com/IQSS/dataverse/issues/10826

Closes #10826 

**Special notes for your reviewer**:

**Suggestions on how to test this**: See new unit test

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**: No

**Is there a release notes update needed for this change?**: No

**Additional documentation**: No

### CODE REVIEW

File: scripts/api/data/metadatablocks/citation.tsv [hunk 1]
```diff
@@ -172,14 +172,13 @@
 	language	Acatepec Me'phaa	tpx	30	tpx
```
Comment (pdurbin) [inline scripts/api/data/metadatablocks/citation.tsv:172]: I'm just putting this comment on the first file...

Do we need a release note? Do we want people to reload the citation block?

**Review by pdurbin — COMMENTED** [review]
Just a quick question.

File: scripts/api/data/metadatablocks/citation.tsv (hunk 1 above)
Comment (pdurbin) [reply scripts/api/data/metadatablocks/citation.tsv:172]: Never mind. In #10762 we already tell people to reload the citation block. This is just a bug fix on top of that PR.

**Review by pdurbin — APPROVED** [review]
The changes seem fine to me, though I'm not an expert. The new unit tests are passing, as are the API tests. Approved.

**Review by qqmyers — APPROVED** [review]
While I'm not a fan of merging codes to avoid the Bundle issue, I don't have any alternatives and the solution here does fix the missing properties issue and, as has been noted, the merges affect only a few rare cases.

**Pull request merged**